### DISCUSSION TITLE
Add Placeholder requirements for Rankups

### DISCUSSION THREAD

**Discussion by goosBanny** [Zrips/CMI#6931] [Enhancement]
Pretty self-explanatory. Could look something like below:

  PlaceholderRequirements:
      - '%statistic_deaths% < 1000'
      - '%statistic_minutes_played% < 10000'

**Comment by mrfloris** [comment]
So much yes. That would make it super dynamic. Upvoted

**Comment by alexey-va** [comment]
Really need this feature too. It will instantly make so many plugins compatible with rankup system

**Comment by ghost** [comment]
+1
